providers/json-binding/src/main/java/org/jboss/resteasy/plugins/providers/jsonb/AbstractJsonBindingProvider.java

```diff
@@ -41,6 +45,12 @@ protected Jsonb getJsonb(Class<?> type) {
    }
 
    public static Charset getCharset(final MediaType mediaType) {
+      System.out.println("== org.jboss.resteasy.jsonb.disable.charset == " + DISABLE_CHARSET);
```

Comment:
@ronsigal This line looks like a typo (should be removed/moved to debug log level)

Reply:
It is a typo indeed. PR updated.

Reply:
@iweiss, what's the status? Are you still having trouble with configuration, and should I be looking at something? Sorry - got distracted.

Reply:
@ronsigal This has been fixed on the latest push. The status for now is that the patch, including the system property, works as expected. The automated Arquillian test, on the other hand, doesn't unless I run the test using Maven with `-Dorg.jboss.resteasy.jsonb.disable.charset=true` set.

Reply:
Ok, @iweiss, I'll play with it.

Reply:
@iweiss, I'm still playing with this test. When I export the WAR, copy it manually to Wildfly, and run the failing tests, it works fine. For some reason, though, the context variable doesn't get set when I run the original Arquillian test. I think there's some Arquillian magic going on that I don't understand. I'll keep trying. 

Btw, I modified AbstractJsonBindingProvider to get the property either from a context variable or a system property. 

-Ron

Reply:
Thanks for the feedback @ronsigal, I'm glad I'm not the only one having issues with Arquillian.